vector_search/utils.py

```diff
@@ -123,12 +161,11 @@ def embed_learning_resources(ids, resource_type):
                 f"{doc['key']}.{doc['run_readable_id']}.{doc['resource_readable_id']}"
             )
         ids.append(vector_point_id(vector_point_key))
-    client.add(
-        collection_name=collection_name,
-        ids=ids,
-        documents=docs,
-        metadata=metadata,
-    )
+    encoder = dense_encoder()
+    embeddings = encoder.encode_batch(docs)
+    vector_name = encoder.model_short_name()
+    points = points_generator(ids, metadata, embeddings, vector_name)
```

Comment:
Would be good to have a unit test to make sure this works as expected (for `create_qdrand_collections` and/or for `points_generator`)

Reply:
👍 tests added for both